lib/broccoli/ember-app.js

```diff
@@ -1845,6 +1845,30 @@ function addOutputFile(container, assetPath, options) {
     container[outputFile] = [];
   }
 
+  if (~container[outputFile].indexOf(assetPath)) {
```

Comment:
@rwjblue always seems to comment on my doing this; it's a habit I picked up and just idiomatic in my head. Rather than have this be a thing I've added us a nice little linting rule to enforce it.